src/components/feed/index.js

```diff
@@ -0,0 +1,44 @@
+import {FaRegComment } from "react-icons/fa"
+import {AiOutlineRetweet} from "react-icons/ai"
+import {MdFavoriteBorder} from "react-icons/md"
+import {VscGraph} from "react-icons/vsc"
+import {LuShare} from "react-icons/lu"
+import { formatDate } from "../../../utilities/formateDate"
```

Comment:
alias paths is configured, update this to use alias path. it's cleaner that way

Reply:
Resolved